index.js

```diff
@@ -234,8 +237,8 @@ module.exports = postcss.plugin('postcss-css-variables', function(options) {
 		rulesThatHaveDeclarationsWithVariablesList.forEach(function(rule) {
 			var rulesToWorkOn = [].concat(rule);
 			// Split out the rule into each comma separated selector piece
-			// We only need to split if is actually comma separted(selectors > 1)
-			if(rule.selectors.length > 1) {
+			// We only need to split if it's actually a Rule with multiple selectors
```

Comment:
Let add "(comma separated)" to end of this comment to add some more context

Reply:
> Let add "(comma separated)" to end of this comment to add some more context